python/ray/scripts/scripts.py

```diff
@@ -759,90 +772,71 @@ def start(
                     cli_logger.print(
                         "the head node: ray start --head --node-ip-address=<public-ip>."
                     )
-                else:
-                    cli_logger.print(
-                        "Multi-node Ray clusters are not supported on OSX and Windows."
-                    )
-                    cli_logger.print(
-                        "If you would like to proceed anyway, restart Ray with:"
-                    )
-                    cli_logger.print(
-                        cf.bold("  ray stop"),
-                    )
-                    cli_logger.print(
-                        cf.bold("  {}=true ray start"),
-                        ray_constants.ENABLE_RAY_CLUSTERS_ENV_VAR,
-                    )
-                cli_logger.newline()
-            else:
-                cli_logger.print("To add another node to this Ray cluster, run")
-                # NOTE(kfstorm): Java driver rely on this line to get the address
-                # of the cluster. Please be careful when updating this line.
-                cli_logger.print(
-                    cf.bold("  ray start --address='{}'"),
-                    bootstrap_address,
-                )
-                cli_logger.newline()
-            if ray_constants.ENABLE_RAY_CLUSTER:
+
+            cli_logger.newline()
+            cli_logger.print(
+                "To connect to this Ray cluster, run `ray.init()` as usual:"
+            )
+            with cli_logger.indented():
+                cli_logger.print("{} ray", cf.magenta("import"))
                 cli_logger.print(
-                    "To connect to this Ray cluster, run `ray.init()` as usual:"
-                )
-                with cli_logger.indented():
-                    cli_logger.print("{} ray", cf.magenta("import"))
-                    cli_logger.print(
-                        "ray{}init()",
-                        cf.magenta("."),
+                    "ray{}init({})",
+                    cf.magenta("."),
+                    "_node_ip_address{}{}".format(
```

Comment:
I added this back because it's needed even for Linux when the node has multiple IPs, this make sure driver and the node use the same ip.